### ISSUE TITLE
Change partialRevealDrawerHeight value

### ISSUE THREAD

**Issue by FilipeAVS** [52inc/Pulley#189]
Hi everyone!

I was wondering if it's possible to **animate** the change to the partialRevealDrawerHeight value while maintaining the drawerPosition at partiallyRevealed.

Currently, when I select a marker, depending on the content shown in the drawer, I set the partialRevealDrawerHeight value. When selecting another marker, I update that same value, and the height changes without any animation.

Thank you!

**Comment by brendan09** [comment]
@FilipeAVS Hello, I just saw your issue this morning. Did you figure it out? Or are you still looking for a solution?

**Comment by FilipeAVS** [comment]
Hello! I actually figured it out! The problem was that I was using
observables to listen for a select on a marker, but a small bug occured
when selecting other marker while still having a previous marker selected.
I was collapsing and revealing the drawer almost at the same time, and a
visual bug occured. I was collapsing the drawer when onDeselect fired on
the MapViewDelegate, but when opening another marker,  onDeselect is called
right before onSelect, collapsing, and then expanding the drawer.

Anyway, I managed to implement a solution and got it working now!

A quinta, 19/04/2018, 16:56, Brendan Lee <notifications@github.com>
escreveu:
